lib/tasks/expiration.rake

```diff
@@ -0,0 +1,21 @@
+require 'iso_8601_duration'
+
+namespace :expiration do
+  desc "Watch for expired variables and rotate"
+  task :watch => :environment do
+
```

Comment:
We should verify that the database is writable before allowing this loop to run.

Reply:
Meaning because we don't want this process running on a follower -- only on a master?   What's the best way to verify this?

Reply:
You can use `SELECT pg_is_in_recovery();` which will return TRUE if the db is in standby and FALSE if not.